samza-api/src/main/java/org/apache/samza/task/TaskContext.java

```diff
@@ -37,6 +37,8 @@
 
   Object getStore(String name);
 
+  Object getTable(String tableId);
```

Comment:
It seems a Table, even with wildcards, would make it more obvious what comes out of this call. At the very least some documentation would help.

Reply:
Fixed